imap_processing/ultra/l1b/ultra_l1b.py

```diff
@@ -0,0 +1,81 @@
+"""Calculates ULTRA L1b."""
+
+import json
+from pathlib import Path
+
+import numpy as np
+import xarray as xr
+
+
+def create_dataset(data_dict, name):
+    """
+    Create xarray for L1b data.
+
+    Parameters
+    ----------
+    data_dict: : dict
+        L1b data dictionary.
+
+    Returns
+    -------
+    dataset : xarray.Dataset
+        xarray.Dataset
+    """
+    # TODO: this will change to actual l1b data dictionary
+    # For now we are using it for retrieving the epoch.
+    dataset = data_dict["imap_ultra_l1a_45de"]
+
+    # Load metadata from the metadata file
+    with open(Path(__file__).parent.parent / "ultra_metadata_example.json") as f:
```

Comment:
I am assuming this file will change now based on yesterday discussion outcome. But if you want to do this update in future PR to keep this thing moving, that sounds good to me.